packages/engine/test/State.test.js

```diff
@@ -18,34 +18,28 @@ import State from '../src/State';
 
 test('set', () => {
   const context = {
-    showValidationErrors: false,
     state: {},
   };
   const state = new State(context);
   state.set('a', 1);
   expect(context.state).toEqual({ a: 1 });
 });
 
-test('resetState and clear showValidationErrors flag', () => {
```

Comment:
We need to remember to test that `block.showValidation` gets set to false  when we reset